src/Core/Service/AppInfo.php

```diff
@@ -15,17 +17,27 @@ public function __construct(
         public string $version,
         public string $hash,
         public string $revision,
-        public string $zipUrl
+        public string $zipUrl,
+        public ?string $hashAlgorithm = null,
+        public ?string $minShopSupportedVersion = null,
```

Comment:
I think we should rename this to `minShopwareSupportedVersion` - and everywhere else it's used.

Reply:
`app-min-shop-supported-version` - this matches the API field. It can refer to both. I don't mind either.

Reply:
I think in the context of Shopware in makes more sense as `shopware` - reading this code without knowledge of the registry you might wonder what it is.